### ISSUE TITLE
Test if newer version break the tests before update

### ISSUE THREAD

**Issue by tunnckoCore** [th0r/npm-upgrade#14]
Probably would be cool if it is an answer, so when you select it, it will run `npm test` (by default) and if it breaks it probably ask you if you are sure to upgrade or not.

I got that idea from `next-update`, but i want to use `npm-upgrade`.

**Comment by tunnckoCore** [comment]
@th0r ping?

**Comment by th0r** [comment]
Yep, idea is nice but unfortunately I don't have time to implement it. PR would be great.

**Comment by olstenlarck** [comment]
Okey, great! I'll see in next weeks :) You can assign me if you want, so I can see this issue easier.
